vars/Archlinux.yml

```diff
@@ -10,6 +10,9 @@ borg_pip_packages:
   - python-pip
   - python-setuptools
 
+borg_docker_packages:
```

Comment:
I didn't review most of this diff, but a comment on this in particular: it's gross and probably Bad(tm) to duplicate this across all the distro var files. The image name is going to be the same regardless of what distro you're on (ignoring the fact that Podman has a different default registry, etc. - but even then, just make the name fully qualified, and it'll be the same, always).

This should probably go into `vars/main.yml` instead.

Reply:
No, they can be different. Like python3-docker, python3.10-docker.

There could be a default, but it needs to be adjustable per-distro.